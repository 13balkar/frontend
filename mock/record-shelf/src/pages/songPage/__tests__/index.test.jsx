import * as React from 'react';
import { render } from '@testing-library/react';
import SongPage from '..';

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('Sync Page', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<SongPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
