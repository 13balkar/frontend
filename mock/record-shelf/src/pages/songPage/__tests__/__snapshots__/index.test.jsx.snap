// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sync Page should render correctly 1`] = `
<DocumentFragment>
  <div
    class="syncPage"
  >
    <div
      class="header"
    >
      <p>
        My 
        <b>
          Record
        </b>
         Shelf
      </p>
    </div>
    <div
      class="cardBlock"
    >
      <p>
        <b>
          all songs
        </b>
         
        <img
          alt="genre"
          src="/assets/icon-genre.svg"
        />
      </p>
      <div
        class="cardList"
      />
    </div>
  </div>
</DocumentFragment>
`;
